### DISCUSSION TITLE
Quite different shape of UMAP plots between scMultiome scRNA-seq and scATAC-seq

### DISCUSSION THREAD

**Discussion by YayanFeng55** [GreenleafLab/ArchR#1884] [Questions / Documentation] [answered]
Hi there,

I have a question about functions of ArchR. Whether ArchR could generate UMAP plots for scRNA-seq ? I have used a scMultiome dataset. I used Seurat to preprocess the scRNA-seq data, and used ArchR to process the matched scATAC-seq data. After labeling the scATAC-seq data with scRNA-seq cell types, I found the two UMAP plots are quite in different shape. I have tried to used Seurat integration pipeline to process my scMultiome data, the UMAP plots are in similar shape. I prefer to use ArchR due to its comprehensive function. So my question is whether ArchR could generate UMAP plots for scRNA-seq ?

**Comment by rcorces** [comment, marked as answer]
ArchR is capable of handling multiome data without pre-processing with Seurat. We are also actively working to support more multiomic work. For now, I'm in the process of updating the multiome documentation and you can see the pre-release version of that documentation here:
https://rcorces.github.io/ArchR_Website_Testing/bookdown/importing-data-and-setting-up-a-multiome-project.html

**Reply by YayanFeng55** [reply]
Hi Ryan, 

Thanks for your quick reply. I also found the documentation from your NG paper: 
https://greenleaflab.github.io/ArchR_2020/Ex-Analyze-Multiome.html
And now I got concordant UMAP plots.

Thanks
Yayan
